.github/workflows/update_doc.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Deploy doc
+on:
+  release:
+    types: [created]
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
```

Comment:
Une image python ne suffit pas ?

Reply:
Ce n'est pas le même fonctionnement que la CI de Gitlab, ici on veut un OS de VM (voir là : https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on)